one-pager-maker/src/App.tsx

```diff
@@ -36,18 +36,29 @@ function App() {
             <BrowserRouter>
                 {user ? (
                     <>
-                        <nav>
-                            <ul id="nav">
-                                <li>
-                                    <Link to="/">一覧</Link>
-                                </li>
-                                <li>
-                                    <Link to="/edit">編集</Link>
-                                </li>
-                                <li>
-                                    <Button onClick={() => { signOut(auth) }}>ログアウト</Button>
+                        <nav className="relative px-4 py-4 flex justify-between items-center bg-white">
```

Comment:
You can make this <nav/> independent, so try moving this to another file.
And this become more natural if you use `header` html tag, instead of nav

Reply:
Move to Header.tsx
header tag